x-pack/test/functional/apps/transform/edit_clone/editing.ts

```diff
@@ -18,9 +18,7 @@ export default function ({ getService }: FtrProviderContext) {
   const esArchiver = getService('esArchiver');
   const transform = getService('transform');
 
-  // FAILING ES PROMOTION: https://github.com/elastic/kibana/issues/208872
-  // FAILING ES PROMOTION: https://github.com/elastic/kibana/issues/208874
-  describe.skip('editing', function () {
+  describe('editing', function () {
```

Comment:
Can these be unskipped too? https://github.com/elastic/kibana/blob/main/x-pack/test/functional/apps/transform/actions/resetting.ts#L17

Reply:
added in https://github.com/elastic/kibana/pull/209109/commits/6b3ed3576026e87c676d372ff907309a89e4b839